code/modules/jobs/job_types/silicon.dm

```diff
@@ -85,5 +85,4 @@ Cyborg
 	return H.Robotize(FALSE, latejoin)
 
 /datum/job/cyborg/after_spawn(mob/living/silicon/robot/R, mob/M)
-	if(CONFIG_GET(flag/rename_cyborg))	//name can't be set in robot/New without the client

```

Comment:
@AnturK Yeah I can remove it from the config. I don't know if the real bug here is that cyborgs can get around it by picking a module but I can't imagine a server wanting to make it so that cyborgs can't name themselves.